File: apps/docs/src/code/python/quickstart.py
from e2b import Sandbox

# 1. Start cloud playground
sandbox = Sandbox()  # $HighlightLine

# 2. Use filesystem
sandbox.filesystem  # $HighlightLine

# 3. Start processes
sandbox.process.start()  # $HighlightLine

# 4. Upload/download files
sandbox.read_bytes()  # $HighlightLine
sandbox.write_bytes()  # $HighlightLine

sandbox.close()

File: apps/docs/src/code/js/quickstart.js
import { Session } from '@e2b/sdk'

// 1. Start cloud playground
const session = await Session.create({ // $HighlightLine
  id: 'Nodejs', // or 'Bash', 'Python3', 'Java', 'Go', 'Rust', 'PHP', 'Perl', 'DotNET'
  apiKey: process.env.E2B_API_KEY,
})

// 2. Use filesystem
session.filesystem // $HighlightLine

// 3. Start processes
session.process.start() // $HighlightLine

// 4. Upload/download files
session.read_bytes() // $HighlightLine
session.write_bytes() // $HighlightLine

await session.close()
